CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 Changelog
 =========
 
+[1.11.2] - 2023-02-20
+--------------------
+
+### New Features
+
+- none
+
+### Bug Fixes
+
+- initscripts: Configure output device in routes
+
+### Other Changes
+
+- Fix CentOS 7 integration tests
+- Refactor route table ID parsing
+- Unify naming of route table name constants
+- ansible-lint: Fix yaml[truthy] failures
```

Comment:
IMHO these internal/non user-visible changes do not need to be mentioned in the changelog.

Reply:
They are in the `Other Changes` section - in the past, we have had similar issues in the past reported in this section of the changelog e.g. https://github.com/linux-system-roles/network/blob/main/CHANGELOG.md#1100---2022-11-01

Reply:
Yes, this happened when I did not review them, I guess. I vaguely remember that I had similar requests in the past.

The changelog for [1.7.0](https://github.com/linux-system-roles/network/blob/main/CHANGELOG.md#170---2022-02-14) does not list all the commits from [1.6.0-1.7.0](https://github.com/linux-system-roles/network/compare/1.6.0...1.7.0) which has similar non-user facing changes. I am fine with accepting this for once but I would like not to have this used as precedent for future entries. This is also an example from the bug fixes section being removed when it was empty.

Reply:
Unfortunately, we cannot currently have empty sections - so what I will do is make `Other Changes` have contents of `- none` - then later we will need to change the way we generate the collection changelog from the individual role changelogs - see https://github.com/linux-system-roles/auto-maintenance/blob/master/release_collection.py#L267